scripts/report-bundle-size/report-as-a-pr-comment.js

```diff
@@ -159,6 +160,22 @@ function formatSize(bundleSize) {
   return `${(bundleSize / 1024).toFixed(2)} kB`
 }
 
+function getEmojiWithThreshold(percentageChange) {
+  const absChange = Math.abs(percentageChange)
+  const emojiMap = [
+    { threshold: 0.15, increase: '😐', decrease: '🙂' },
+    { threshold: 0.3, increase: '😠', decrease: '😀' },
+    { threshold: 0.5, increase: '😡', decrease: '😃' },
+    { threshold: Infinity, increase: '⛔', decrease: '🤩' },
+  ]
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏any reasoning on the threshold values and the number of thresholds?

Reply:
Sorry, I just updated the values to 1.5 / 3.0 / 4.5. We discussed the values with Alfonso after calculating what 1 %, 3 %... would represent in terms of bytes. Do you have an other approach to decide those ?

Reply:
IMO, it could be interesting that the signal raised has a clear meaning and push us to do something.
Should we react differently between a 😠 and a 😡?
Maybe having a single threshold for significant increase/decrease could be enough.

About threshold values, I think displaying % is valuable in the report but using % for thresholds is probably not ideal, since it would allow bigger and bigger increases as the package size is growing.